src/pages/subreddit/Subreddit.js

```diff
@@ -0,0 +1,42 @@
+import React, {useEffect, useState} from 'react';
+import './Subreddit.css';
+import {Link} from 'react-router-dom';
+import {useParams} from "react-router-dom";
+import axios from "axios";
+
+function Subreddit() {
+    const {subredditId} = useParams();
+    const [post, setPost] = useState([]);
```

Comment:
Als `post` uiteindelijk een object zal zijn in plaats van een array, dan zou ik als default waarde `undefined` pakken
```suggestion
    const [post, setPost] = useState();
```